appengine/mailjet/app.js

```diff
@@ -0,0 +1,86 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var express = require('express');
+var path = require('path');
+var bodyParser = require('body-parser');
+
+// [START setup]
+var Mailjet = require('node-mailjet').connect(
+  process.env.MJ_APIKEY_PUBLIC,
+  process.env.MJ_APIKEY_PRIVATE
+);
+// [END setup]
+
+var app = express();
+
+// Setup view engine
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'jade');
+
+// Parse form data
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+// [START index]
+app.get('/', function(req, res) {
+  res.render('index');
+});
+// [END index]
+
+// [START hello]
+app.post('/hello', function(req, res, next) {
+  var options = {
+    // From
+    FromEmail: 'no-reply@appengine-mailjet-demo.com',
```

Comment:
Mmmmmkay. SGTM.

On Tue, Apr 19, 2016 at 1:10 PM Jason Dobry <notifications@github.com>
wrote:

> In appengine/mailjet/app.js
> <https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/102#discussion_r60302073>
> :
>
> > +
> > +// Parse form data
> > +app.use(bodyParser.json());
> > +app.use(bodyParser.urlencoded({ extended: false }));
> > +
> > +// [START index]
> > +app.get('/', function(req, res) {
> > +  res.render('index');
> > +});
> > +// [END index]
> > +
> > +// [START hello]
> > +app.post('/hello', function(req, res, next) {
> > +  var options = {
> > +    // From
> > +    FromEmail: 'no-reply@appengine-mailjet-demo.com',
>
> As long as they've verified that email address it will send.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/102/files/4f14285df4f97655537b8770fa34c44949c4aafc#r60302073>
>